File: students/students/src/MON.SPA/src/models/studentSummaryModel.js
import {StudentClassSummaryModel} from '@/models/studentClass/studentClassSummaryModel.js';
export class StudentSummaryModel {
    constructor(obj = {}) {
      this.personId = obj.personId;
      this.fullName = obj.fullName || '';
      this.gender = obj.gender || '';
      this.pin = obj.pin || '';
      this.pinType = obj.pinType || '';
      this.age = obj.age || 0;
      this.toBeDischarged = obj.toBeDischarged || false;
      this.currentClass = new StudentClassSummaryModel(obj.currentClass);
      this.allCurrentClasses = obj.allCurrentClasses || [];
      this.waitingToBeDischarged = obj.waitingToBeDischarged || [];
      this.azureAccountDeleted = obj.azureAccountDeleted;
      this.email = obj.email;
      this.mobilePhone = obj.mobilePhone;
    }
  }

File: students/students/src/MON.Models/StudentModels/StudentSummaryModel.cs
using System.Collections.Generic;

namespace MON.Models.StudentModels
{
    public class StudentSummaryModel : StudentSummaryBaseModel
    {
        public string Gender { get; set; }
        public string PIN { get; set; }
        public string PINType { get; set; }
        public int PintTypeId { get; set; }
        public int Age { get; set; }
        public bool AzureAccountDeleted { get; set; }
        public StudentClassSummaryModel CurrentClass { get; set; }
        public List<StudentClassSummaryModel> AllCurrentClasses { get; set; }
        public List<WaitingToBeDischarged> WaitingToBeDischarged { get; set; }
        public string Email { get; set; }
        public string MobilePhone { get; set; }
    }
}
